ShoppingList/src/main/java/org/openintents/shopping/ui/widget/ShoppingItemsView.java

```diff
@@ -121,7 +121,7 @@
 
     public int mMode = ShoppingActivity.MODE_IN_SHOP;
     private String mFilter;
-    private boolean mInSearch;
+    public boolean mInSearch;
```

Comment:
Please use a getter instead of direct field access.

Reply:
Done